docs/tutorials/offline-dev.md

```diff
@@ -16,14 +16,44 @@ To enable offline custom TLD goodness, we'll use DNSMasq to route traffic to our
 address=/dev/127.0.0.1
 ```
 Sub in your favorite TLD for 'dev' here if you want to use something more flashy.
-5. Restart DNSMasq: `brew services restart dnsmasq`
+5. Restart DNSMasq: `sudo brew services restart dnsmasq`  ( make sure you use `sudo` as other users are not able to change the mappings ).
 6. Open your `~/.lando/config.yml` file and add the following line:
 ```yaml
 proxyDomain: dev
 ```
 Again, allow your creativity to flourish and use whatever TLD makes you smile.
-7. Power Lando down with `lando poweroff`
-8. Start up your favorite Lando app to test it out with `lando start` from within your app's root directory. You should see something like:
+
+7. Notify macOS that you are using a local DNS server by adding it to your Network configuration.
+
+Assuming you are on a Wifi Network you would run the following :
+
+```bash
+networksetup -setdnsservers Wi-Fi 127.0.0.1 192.168.0.1
```

Comment:
I'd like some more confirmation that this is necessary. I did not have to go through this process. Is it High Sierra specific?